### DISCUSSION TITLE
part-ii/2011-130/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#176] [Question comments]
Part II 2011 Riemann Surfaces: Paper 1, Section II, 23G 

https://questions.tripos.org/part-ii/2011-130/

**Comment by silkca** [comment]
For the first two parts, working in charts and using results from Complex Analysis yields the results.

For the last part, there are obvious ways of writing $\alpha$ and $\alpha^{-1}$ in terms of functions we know are analytic, at which point the earlier parts can be applied directly, and the example asked for is trivial using a result about the conformal equivalences of complex tori.
